bot/subcommand.go

```diff
@@ -390,6 +390,30 @@ func (sub *Subcommand) SetPlumb(methodName string) {
 	sub.plumbed = sub.FindCommand(methodName)
 }
 
+// AddAliases add alias(es) to specific command (defined with MethodName).
+// This removes silently all duplicate aliases. Use this instead overwriting aliases
+// for safety reasons (duplicates).
+func (sub *Subcommand) AddAliases(commandName string, aliases ...string) {
+	// Get command
+	command := sub.FindCommand(commandName)
+
+	// Merge current aliases for duplicate removing
+	aliases = append(aliases, command.Aliases...)
+
+	// Remove all duplicate aliases
+	keys := make(map[string]bool)
```

Comment:
This seems kind of excessive. I think it would be fine to leave it to the developer.